src/chunks/vocab/mod.rs

```diff
@@ -0,0 +1,121 @@
+//! Embedding vocabularies
+
+use std::collections::HashMap;
+use std::io::{Read, Seek, Write};
+
+use byteorder::{LittleEndian, ReadBytesExt, WriteBytesExt};
+
+use crate::io::{Error, ErrorKind, Result};
+
+mod subword;
+pub use subword::{
+    BucketSubwordVocab, ExplicitSubwordVocab, FastTextSubwordVocab, NGramIndices, SubwordIndices,
+    SubwordVocab,
+};
+
+mod simple;
+pub use simple::SimpleVocab;
+
+mod wrapper;
```

Comment:
We call the module `wrappers` for storage, so we should do the same here.

Reply:
There it's actually multiple wrappers, `StorageWrap` and `StorageViewWrap`. If you really don't like it, I can change it

Reply:
Yep, naming consistency between the modules is nice.